census_blueprints/verify_sync_status.py

```diff
@@ -0,0 +1,115 @@
+import sys
+import argparse
+import requests
+import shipyard_utils as shipyard
+
+
+EXIT_CODE_STATUS_COMPLETED = 0
+EXIT_CODE_INVALID_CREDENTIALS = 200
+EXIT_CODE_BAD_REQUEST = 201
+EXIT_CODE_UNKNOWN_ERROR = 211
+EXIT_CODE_SYNC_CHECK_ERROR = 220
+EXIT_CODE_STATUS_RUNNING = 222
+EXIT_CODE_STATUS_FAILED = 223
+
```

Comment:
For consistency across other Blueprints, these should be named:

EXIT_CODE_FINAL_STATUS_SUCCESS = 0
EXIT_CODE_STATUS_INCOMPLETE = 210
EXIT_CODE_FINAL_STATUS_ERRORED = 211

Might have to change some other errors/error codes as a result. I've generally found that 200-209 is great for vendor authorization type errors and 210-219 is great for status checking errors.